deepface/detectors/OpenCv.py

```diff
@@ -168,6 +168,9 @@ def __get_opencv_path(self) -> str:
         Returns:
             installation_path (str)
         """
+        if os.getenv("OPENCV_PATH", "") != "":
```

Comment:
Prefer to get opencv path dynamically instead of env vars. Would you please revert this?